packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -399,4 +406,17 @@ public void setMinMaxZoomPreference(Float min, Float max) {
   public void setZoomGesturesEnabled(boolean zoomGesturesEnabled) {
     googleMap.getUiSettings().setZoomGesturesEnabled(zoomGesturesEnabled);
   }
+
+  @Override
+  public void setMyLocationEnabled(boolean myLocationEnabled) {
+    this.myLocationEnabled = myLocationEnabled;
+    if (googleMap != null) {
+      if (ContextCompat.checkSelfPermission(context, Manifest.permission.ACCESS_FINE_LOCATION)
```

Comment:
We definitely should fail here, I filed flutter/flutter#24327 to refactor the options update to be fail-able, for now lets at least `Log.e` an error message, you can leave a:
```
// TODO(amirh): Make the options update fail. 
// https://github.com/flutter/flutter/issues/24327
```